roles/network/defaults/main.yml

```diff
@@ -119,6 +119,10 @@ network_dispatcher_scripts: []
 #   - src: /opt/configuration/network/iptables.sh
 #     dest: routable.d/iptables.sh
 
+network_networkd_path: "/etc/systemd/network"
```

Comment:
for which use case does this need to be configurable?

Reply:
I actually just modelled this after [what is configurable for netplan](https://github.com/osism/ansible-collection-commons/blob/main/roles/network/defaults/main.yml#L93).
I have to admit that I cannot think of a use case, so I will just remove it